proposals/csharp-10.0/caller-argument-expression.md

```diff
@@ -148,6 +148,7 @@ There should always be an expression corresponding to the `this` parameter. Even
 - Multiple parameters marked with `CallerArgumentExpression` are permitted, as shown above.
 - The attribute's namespace will be `System.Runtime.CompilerServices`.
 - If `null` or a string that is not a parameter name (e.g. `"notAParameterName"`) is provided, the compiler will pass in an empty string.
+- The type of the parameter `CallerArgumentExpressionAttribute` is applied to must have a standard conversion from `string`. This means no user-defined conversions from `string` are allowed, and in practice means the type of such a parameter must be `string` or `object`.
```

Comment:
Any interface implemented by `string` will work, also `dynamic` will work. So it's not only `string` and `object`.

Reply:
Side note (out of scope for this PR): The other attributes error messages (and also the spec) mention "standard implicit conversion" instead of "standard conversion", which is wrong. This was pointed out by Aleksey in the code review IIRC.

Reply:
Does "user-defined conversions" mean `ReadOnlySpan<char>` is also out of the question?

Reply:
Thanks for the clarification @Youssef1313.

> Does "user-defined conversions" mean ReadOnlySpan<char> is also out of the question?

Yes, it does mean that.